pkg/cli/config/config.go

```diff
@@ -50,6 +50,7 @@ type BackplaneConfiguration struct {
 	DisplayClusterInfo          bool                            `json:"display-cluster-info"`
 	DisableKubePS1Warning       bool                            `json:"disable-kube-ps1-warning"`
 	Govcloud                    bool                            `json:"govcloud"`
+	SessionDurationMinutes      int                             `json:"session-duration-minutes"`
```

Comment:
Shall we set up a default value for the `SessionDurationMinutes` ?

Reply:
I rely on default of 0 with the logic. I made it simpler realying on the fact that default in AWS is 15 minutes, and we want to input larger times, not smaller. 

Should be much clearer now.